src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -248,7 +253,7 @@ class EduSharingConnector {
 				values: ['1'],
 			});
 		} else if (collection) {
-			criterias.push({ property: 'ngsearchword', values: ['*'] });
+			criterias.push({ property: 'ngsearchword', values: [''] });
```

Comment:
for consistency / readability, this can also be renamed to criteria

Reply:
The EduSharingConnector will be merged from main after PR https://github.com/hpi-schul-cloud/schulcloud-server/pull/3810 is done. So no review needed at this point in first place. Sorry for the trouble.